File: app/src/main/java/com/dmi/perfectreader/render/RenderSpace.java
package com.dmi.perfectreader.render;

import android.graphics.Canvas;

import com.dmi.perfectreader.style.FontStyle;

import java.util.Locale;

public class RenderSpace extends RenderText {
    private float scaleX;

    public RenderSpace(float width, float height, CharSequence text, Locale locale, float baseline, float scaleX, FontStyle style) {
        super(width, height, text, locale, baseline, style);
        this.scaleX = scaleX;
    }

    public float scaleX() {
        return scaleX;
    }

    @Override
    public void paintItself(Canvas canvas) {
    }
}

File: app/src/main/kotlin/com/dmi/perfectreader/render/RenderSpace.kt
package com.dmi.perfectreader.render

import android.graphics.Canvas
import com.dmi.perfectreader.location.BookRange
import com.dmi.perfectreader.style.FontStyle
import java.util.*

class RenderSpace(width: Float,
                  height: Float,
                  text: CharSequence,
                  locale: Locale,
                  baseline: Float,
                  style: FontStyle,
                  val scaleX: Float,
                  range: BookRange
) : RenderText(width, height, text, locale, baseline, style, range) {
    override fun paintItself(canvas: Canvas) = Unit
}
